app/controllers/reacts_controller.rb

```diff
@@ -5,13 +5,22 @@ def create
     return if reacted?(params[:post_id])
 
     @react = React.new(user_id: current_user.id, post_id: params[:post_id])
-    @react.save
+    respond_to do |format|
+      if @react.save
+        format.js 
+      end
```

Comment:
In this case, how would you do if you want to response when the save is failing?
`Respond_to` should be use when you want to return the response.